_episodes_rmd/02-r-basics.Rmd

```diff
@@ -493,22 +497,22 @@ Notice in the operation above that R inserts an `NA` value to extend our vector
 > Answer the following questions to test your knowledge of vectors
 >
 > Which of the following are true of vectors in R?
-> A) All vectors have a mode or a length
-> B) All vector have a mode and a length
-> C) Vectors may have different lengths
-> D) Items within a vector may be of different modes
-> E) You can use the `c()` to one or more items to an existing vector
-> F) You can use the `c()` to add a vector to an exiting vector
+> A) All vectors have a mode or a length  
+> B) All vector have a mode and a length  
```

Comment:
vector -> vectors

Maybe make the and and the or bold or italic to make them stand out as the difference between A and B?